### PR TITLE
feat(t8s-cluster/raise_api_version): raise api version for helm.toolk…

### PR DESCRIPTION
**Pull request by marvinWolff**
…it.fluxcd.io

### CODE REVIEW

**Review by tasches — DISMISSED** [review]

**Comment by marvinWolff** [comment]
@cwrau the failing check is in regards to too much parallelization, i guess?

File: charts/t8s-cluster/templates/workload-cluster/_helmRelease.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {{- define "t8s-cluster.helm.resourceIntoCluster" -}}
   {{- $_ := mustMerge . (pick .context "Values" "Release" "Chart") -}}
-apiVersion: helm.toolkit.fluxcd.io/v2beta2
```
Comment (cwrau) [inline charts/t8s-cluster/templates/workload-cluster/_helmRelease.yaml:3]: Why did you delete this?

Comment (marvinWolff) [reply charts/t8s-cluster/templates/workload-cluster/_helmRelease.yaml:3]: Oh. Not on purpose.

Guess this was a typo in vim.

Comment (marvinWolff) [reply charts/t8s-cluster/templates/workload-cluster/_helmRelease.yaml]: Fixed it.

**Review by cwrau — APPROVED** [review]
